sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/NetAppVault.cs

```diff
@@ -1,35 +1,35 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// <auto-generated/>
-
-#nullable disable
-
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
 using Azure.Core;
 using Azure.ResourceManager.Models;
 
 namespace Azure.ResourceManager.NetApp.Models
 {
     /// <summary> Vault information. </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
     public partial class NetAppVault : ResourceData
     {
-        /// <summary> Initializes a new instance of NetAppVault. </summary>
-        internal NetAppVault()
-        {
-        }
+        /// <summary> Vault Name. </summary>
+       // [EditorBrowsable(EditorBrowsableState.Never)]
+        public string VaultName { get; }
```

Comment:
Could we have a format, clean up and reordering of its elements to minimize the diff in this file?
We could have just two changes in this file: adding EBNever attribute on this class and removing the `auto-generated` comment label on line 4